### DISCUSSION TITLE
Release for Windows 10 32-bit

### DISCUSSION THREAD

**Discussion by gencho-kalaykov** [dorssel/usbipd-win#697] [Ideas]
Is it possible to compile release for Windows 10 32bit? I need it for specific software running only on 32-bit platform. Thank you in advance!

**Comment by dorssel** [comment]
`usbipd-win` is server side only.

> ...specific software running on 32-bit platform...

This sounds like client-side.

If you really mean server side: that's not going to happen. For the same reason as #486. `usbipd-win` relies on drivers from VirtualBox, which only exist for x64 platforms. See https://forums.virtualbox.org/viewtopic.php?t=95557.

**Comment by gencho-kalaykov** [comment]
Yes, we need client side only! But the installer is for x64 and I cannot
run it on my Win10 x86 machine. I found Windows Client v.0.2.0.0 Patched
that doesn't need an installer and has x86 and x64 drivers for manual
installation. But it doesn't bind to a linux server. It attaches to your
usbipd-win on Win10 x64, but I'd prefer to have my USB dongles on a linux
machine.

https://sourceforge.net/p/usbip/discussion/418507/thread/86c5e473/

**Reply by dorssel** [reply]
Again: `usbipd-win` is server side only. You'll have to look elsewhere for clients. Look through older discussions and issues. Some people have made it work with Windows clients, but I'm not sure about x86.
